new-lamassu-admin/src/pages/MachineLogs.js

```diff
@@ -134,6 +134,10 @@ const Logs = () => {
                 ))}
             </TableBody>
           </Table>
+          {loading && <H4>{'Loading...'}</H4>}
+          {!loading && !R.isEmpty(logsResponse) && (
```

Comment:
`!R.isEmpty(logsResponse)` isn't this going to show "No activity so far" only when the table has data? Also the logs are in `logsResponse.machineLogs` so the empty test should be of this.